### PR TITLE
Extend spec. Docs review

### PR DESCRIPTION
**Pull request by savq**
- Add capitals to comments to differentiate them more from the code
- Simplify a couple of sentences
- Fix small typos

### CODE REVIEW

File: doc/lush.txt [hunk 1]
```diff
@@ -361,42 +361,41 @@ extends({parsed_lush_spec, ...}).with(spec)                  *lush-extends-with*
...
-  -- maybe you also want to include a community extension that patches
-  -- some mising styles in the original theme.
+  -- Maybe you also want to include a community extension that patches
+  -- some missing styles in the original theme.
   local some_theme_lsp = require('lush_theme.some_theme_lsp')
 
   local spec = lush.extends({some_theme, some_theme_lsp}).with(function()
-    -- note it is also valid to return an empty spec, which is functionally
-    -- equivilent to calling merge({some_theme}).
+    -- It's also valid to return an empty spec. That's functionally
```
Comment (rktjmp) [inline doc/lush.txt:386]: > It's also valid to return an empty spec. That's functionally equivalent to calling merge({some_theme}).

I think I prefer the more declarative "It is also valid to".

Also "That's" isn't wrong but I thing its a bit awkward. Hard to express why, it's almost like that's is past tense. 

Probably

> "This is functionally equivalent calling merge({some_theme})."

 reads better *and* connects better to the `return {}`?

File: doc/lush.txt [hunk 2]
```diff
@@ -424,34 +423,33 @@ Potential use cases:
...
   if user_config.enable_xyz then
     table.insert(specs, require('lush_theme.some_theme.xyz')
   end
 
-  -- you might also include variants this way
+  -- You might also include variants this way
   it user_config.italic_comments then
-    -- you can also do this inline, or this could also
-    -- be done with extends.with
+    -- You can also do this inline, or with extends.with
```
[the remark this answers is not in the document]
Comment (savq) [reply doc/lush.txt:445]: Perhaps "or using extends.with"?

File: doc/lush.txt (hunk 1 above)
Comment (savq) [reply doc/lush.txt:386]: Yes, that also works.

File: doc/lush.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (rktjmp) [reply doc/lush.txt:445]: Yep, that's what I ended up on :+1:

**Comment by rktjmp** [comment]
Thanks for the help.

**Pull request merged**